### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
project live - https://solutionground.netlify.app/

### CODE REVIEW

File: navigation/Navigation/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
+@import "../../utility/component.css";
+
+/* .horizontal-nav-header{
+    display: flex;
+}
+
+.horizontal-nav-list{
+  display: flex;
+} */
```
Comment (Mohasin-Haque) [inline navigation/Navigation/Navigation.css]: remove this

File: navigation/utility/utility-icon-size.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  .icon-size-mediun{
+    font-size: 2rem !important;
+  }
+  
+  .icon-size-small{
+    font-size: 1rem !important;
+  }
+  
+  .icon-size-extrasmall{
+    font-size: 0.5rem !important;
```
Comment (Mohasin-Haque) [inline navigation/utility/utility-icon-size.css]: avoid using important

**Review by Mohasin-Haque — COMMENTED** [review]
Your code is god but it needs to be formatted and clean. It will be good if you remove the whitespaces and add meaningful comments!! Also do remove !important and try to solve it by specificity.

File: navigation/Navigation/Navigation.css (hunk 1 above)
Comment (chanchal-panpaliya) [reply navigation/Navigation/Navigation.css]: done

File: navigation/utility/utility-icon-size.css (hunk 2 above)
Comment (chanchal-panpaliya) [reply navigation/utility/utility-icon-size.css]: done

**Review by HrishikeshMahalle — APPROVED** [review]
Code looks fine... Html is really well presented.

**Pull request merged**